services/credential-server/src/modules/signify/signifyApi.ts

```diff
@@ -74,7 +74,7 @@ export class SignifyApi {
 
   async resolveOobi(url: string): Promise<any> {
     const alias = new URL(url).searchParams.get("name") ?? uuidv4();
-    const operation = await waitAndGetDoneOp(
+    const operation: any = await waitAndGetDoneOp(
```

Comment:
There should be an operation type from signify

Reply:
I have to use type any because type of property response in Operation is unknow, so i still have type error at operation.response.i